packages/saved-views-react/CHANGELOG.md

```diff
@@ -6,6 +6,43 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased](https://github.com/iTwin/saved-views/tree/HEAD/packages/saved-views-react)
 
+### Beaking changes
+
+* `SavedView` interface changes
+  * Rename `id` property to `savedViewId`
+  * Remove `thumbnail` property
+  * Move `viewData` and `extensions` properties `SavedViewData` type
+    * Type of `viewData` has been changed and definition has moved to `@itwin/saved-views-react`
+  * Change type of `creationTime` and `lastModifier` properties to `Date | undefined`
```

Comment:
```suggestion
  * Change type of `creationTime` and `lastModified` properties to `Date | undefined`
```

Reply:
Fixed in b892f140abfcddf4289c5543f72b4bbbc9babfd9.